workspaces/adoption-insights/plugins/adoption-insights-backend/src/controllers/EventApiController.ts

```diff
@@ -25,30 +25,40 @@ import { toEndOfDayUTC, toStartOfDayUTC } from '../utils/date';
 import { EventSchema } from '../validation/event';
 import { EventRequestSchema } from '../validation/event-request';
 import { ValidationError } from '../validation/ValidationError';
-import { RootConfigService } from '@backstage/backend-plugin-api';
+import {
+  AuditorService,
+  AuditorServiceEvent,
+  RootConfigService,
+} from '@backstage/backend-plugin-api';
 import { getLicensedUsersCount } from '../utils/config';
 import { TechDocsCount, TopTechDocsCount } from '../types/event';
 
 class EventApiController {
```

Comment:
This class also has  [getInsights]( https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/adoption-insights/plugins/adoption-insights-backend/src/controllers/EventApiController.ts#L81) method for retrieving insights, can we log that as well ?